app/assets/stylesheets/application.css.scss

```diff
@@ -3,6 +3,8 @@
 
 @import "normalize";
 
-@import "daplog/_base";
-@import "daplog/_blocks";
-@import "daplog/_contexts";
+@import "daplog/base";
+@import "daplog/blocks";
+@import "daplog/contexts";
+
+.foo{color:red}
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Selector `foo` should be written must match regex /^([A-Z]|u-|is-)[-a-zA-Z0-9]*$/<br>Colon after property should be followed by one space<br>Opening curly brace in a single line rule set `{` should be preceded by at least one space<br>Declaration should be terminated by a semicolon